docs/local-setup/create-account.md

```diff
@@ -48,24 +48,22 @@ Once you click on the "PROTECT ACCOUNT" button, you will be forwarded to the das
 
 Alternatively, you can choose to backup your private keys manually. To do so click below the the phone number entry on "I choose to backup my account manually". In this case we're not backing up "private keys". Instead, we are guaranteeing that your account can be recovered using a different (ie. not phone SMS) method; in our case, by using a seed phrase.
 
-In the next step, you will be presented with a seed phrase that will allow you to recover your key pair in the future. 
+In the next step, you will be presented with a seed phrase that will allow you to recover your key pair in the future.
 
 ![SeedPhrase](assets/writedownrecovery.png)
 
 **Important** Please write down the seed phrase in a secure place, potentially make a copy and store both copies at two different places. We will not have a backup and will not be able to help you recover your key pair. Nevertheless, properly backing up your key manually is normally the more secure option.
 
-Once you click "Setup seed phrase", you will be directed to your wallet interface, so please make sure that you have taken note of your seed phrase beforehand. 
+Once you click "Setup seed phrase", you will be directed to your wallet interface, so please make sure that you have taken note of your seed phrase beforehand.
 
 ![Wallet](assets/finalscreen.png)
 
 ### Near key recovery
```

Comment:
@amgando We're removing this functionality very shortly, and will no longer be able to assist in key recovery (which is a win for security and not being a custodian!) I'll provide more details on the new mechanisms ASAP.

In the meantime, we should consider dropping this section entirely.

Reply:
will remove

Reply:
✅